src/test/java/RoleServiceImplTest.java

```diff
@@ -0,0 +1,19 @@
+import com.mateacademy.springmvc.entity.Role;
+import com.mateacademy.springmvc.repository.RoleRepository;
+import com.mateacademy.springmvc.service.RoleServiceImpl;
+import org.junit.Test;
+import static org.mockito.Mockito.verify;
+
+public class RoleServiceImplTest {
```

Comment:
Is this test green? I think no. 
If you did not mock RoleRepository it would not work. 
Please fix it.